_im-plugin/ism/policies.md

```diff
@@ -162,6 +163,46 @@ Parameter | Description | Type | Required
 
 For information about setting replicas, see [Primary and replica shards]({{site.url}}{{site.baseurl}}/opensearch#primary-and-replica-shards).
 
+### shrink
+
+Allows you to reduce the number of primary shards in your indexes. With this action, you can specify:
+
+- the number of primary shards that the target index should contain.
+- A max shard size for the primary shards in the target index.
+- Specify a percentage to shrink the number of primary shards in the target index.
+
+```json
+"shrink": {
+    "num_new_shards": {
+        "type": "integer"
+    },
+    "max_shard_size": {
+        "type": "keyword"
+    },
+    "percentage_of_source_shards": {
+        "type": "double"
+    },
+    "target_index_suffix": {
+        "type": "text"
+    },
+    "aliases": {
+        "type": "object"
+    },
+    "force_unsafe": {
+        "type": "boolean"
+    }
+}
+```
+
+Parameter | Description | Type | Example | Required
+:--- | :--- |:--- |:--- |
+`num_new_shards` | The maximum number of primary shards in the shrunken index. | integer | `5` | Yes, however cannot be used with `max_shard_size` or `percentage_of_source_shards`
+`max_shard_size` | The maximum size in bytes of a shard for the target index. | keyword | `5gb` | Yes, however cannot be used with `max_shard_size` or `percentage_of_source_shards`
+`percentage_of_source_shards` | Percentage of the number of original primary shards that will be shrunk. This indicates the minimum percentage by which the number of primary shards will shrink. Must be between 0.0 and 1.0 exclusive.  | Percentage | `0.5` | Yes, however cannot be used with `max_shard_size` or `percentage_of_source_shards`
+`target_index_suffix` | Suffix that will be appended to the target index to by shrunk. | text | `shrunken` | No
+`aliases` | Aliases which will be added to the new index. | object | `myalias` | No
```

Comment:
More future tense here. Also, is text the parameter type? Can we use string?

Reply:
It is object type, but is an array of alias objects. This is a fairly convoluted field so I am adding a large comment on the PR with a few examples.